AirCasting/SessionViews/ReconnectSessionCardViewModel.swift

```diff
@@ -26,6 +44,11 @@ class ReconnectSessionCardViewModel: ObservableObject {
         }
     }
     
+    // This is just in case something goes wrong and the viewModel stays in memory
```

Comment:
I still don't get why this would happen 🤷‍♂️

Reply:
this method will be triggered every time the view refreshes or the tab is changed to another and back again. Do we need it at all?

Reply:
True, okay, I'll remove it